src/pyscheduling/JS/JmCmax.py

```diff
@@ -283,7 +301,8 @@ def shifting_bottleneck(instance : JmCmax_Instance):
         """
         startTime = perf_counter()
         solution = JobShop.JobShopSolution(instance)
-        graph = JobShop.Graph(instance.P)
+        #graph = JobShop.Graph(instance)
+        graph = JobShop.JobsGraph(instance)
```

Comment:
The JobsGraph object should be created from the solution, is there a way you can do that ?